app/src/main/java/com/keylesspalace/tusky/usecase/DeveloperToolsUseCase.kt

```diff
@@ -0,0 +1,46 @@
+package com.keylesspalace.tusky.usecase
+
+import android.util.Log
+import androidx.room.withTransaction
+import com.keylesspalace.tusky.db.AppDatabase
+import com.keylesspalace.tusky.db.TimelineDao
+import javax.inject.Inject
+
+/**
+ * Functionality that is only intended to be used by the "Developer Tools" menu when built
+ * in debug mode.
+ */
+class DeveloperToolsUseCase @Inject constructor(
```

Comment:
ProGuard already removes it for release builds, because in a release build there's no code that calls it.

You can verify this by:

1. Perform a release build
2. Run `grep DeveloperTools app/build/outputs/mapping/blueRelease/usage.txt` or similar

Despite the `usage.txt` name, this file is a list of all the things that ProGuard removed, and you can confirm that the `DeveloperTools` class, the `MainActivity.buildDeveloperToolsDialog()` function, the companion object, the factory class used for injection, etc etc, are all listed in this file, so they're not in the APK.

Reply:
Ok, I see, thank you